handbook/engineering/search/iteration_log.md

```diff
@@ -14,6 +14,14 @@ This document contains the goals and work log for the search team's [2-week iter
     
 ## 2020-12-14 to 2021-01-04
 
+### Streaming search
+- **Owner(s):** Juliana, Keegan
+- **Outcomes:**
```

Comment:
@keegancsmith  do you want to add any backend goals you have plans on working on?

Reply:
added